json/entities/entities.schema.json

```diff
@@ -7,122 +7,111 @@
+              "loc.adm.nat",
+              "loc.adm.sup",
+              "loc.phys.geo",
+              "loc.phys.hydro",
+              "loc.phys.astro",
+              "loc.oro",
+              "loc.fac",
+              "loc.add.phys",
+              "loc.add.elec",
+              "loc.unk",
+              "comp.name",
+              "comp.title",
+              "comp.qualifier",
+              "comp.function",
+              "comp.demonym"
             ]
           },
           "surface": {
             "type": "string",
-            "description":"The (string) surface of the named entity mention, as it appears in the text"
-          },
-          "name": {
-            "type": "string",
-            "description":"In case of a person mention, the entity component of type name."
+            "description": "The surface form of the named entity mention, as it appears in the text."
           },
           "lOffset": {
             "type": "integer",
-            "description":"The left character offset of the named entity with respect to the content item, as in the rebuilt format."
+            "description": "The left character offset of the named entity with respect to the content item."
           },
           "rOffset": {
             "type": "integer",
-            "description":"The right character offset of the named entity with respect to the content item, as in the rebuilt format."
-          },          
-          "firstname": {
-            "type": "string",
-             "description":"In case of a person mention and if available, the first name."
+            "description": "The right character offset of the named entity with respect to the content item."
           },
-          "surname": {
-            "type": "string",
-             "description":"In case of a person mention and if available, the surname."
+          "nested": {
+            "type": "boolean",
+            "description": "Indicates whether the mention is nested."
           },
-          "title": {
-            "type": "string",
-            "description":"In case of a person mention, the entity component of type 'title'."
+          "confidence_ner": {
+            "type": "number",
+            "description": "Confidence score of the Named Entity Recognition process."
           },
-          "function": {
+          "confidence_nel": {
+            "type": "number",
+            "description": "Confidence score of the Named Entity Linking process."
+          },
+          "wkd_id": {
             "type": "string",
-            "description":"In case of a person mention, the entity component of type 'function'."
+            "description": "Wikidata QID if available."
           },
-          "demonym": {
+          "wkpedia_pagename": {
             "type": "string",
-            "description":"In case of a person mention, the entity component of type 'demonym'."
+            "description": "Wikipedia page name or URL in the relevant language."
```

Comment:
Originally the mix of pagename and URL as accepted values comes from a certain flexibility we had, and from the fact that we were actually never using the URL (meant as a fallback).

This schema revision could be the opportunity to correct into:
```
 "wkpedia_pagename": {
            "type": "string",
            "description": "Wikipedia page name, i.e. the last part of the wikipedia URL (e.g.  United_States)"
    },      
            
   "wkpedia_url": {
            "type": "string",
            "description": "Wikipedia page URL, e.g. https://en.wikipedia.org/wiki/United_States"
    }
```

The pagename itself is not really telling since without the language, one cannot reconstruct the URL. However we are using only the wkd_id to link to external resource, and the pagename is used as part of a unique identifier.

If one day we want/need to use the URL, it is best to have a proper field to store it I think. What do you think?

Reply:
Agree. Changes made (even before seeing the review!)